handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -132,8 +137,22 @@
                             }
                             availableOpenSslCipherSuites.add(c);
                         }
+                        try {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Extract this nested try block into a separate method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1141)